docs/detections/rules-ui-create.asciidoc

```diff
@@ -289,7 +289,11 @@ TIP: Click the help icon (image:images/esql-ref-button.png[Click the ES|QL help
 [[esql-agg-query]]
 ===== Aggregating query 
 
-Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Alerts generated by an {esql} rule with an aggregating query only contain the fields returned by the query.
+Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Alerts generated by a rule with an aggregating query only contain the source event fields that the {esql} query returns and new fields that the query creates.
+
+TIP: To improve the visibility of these fields while investigating alerts that the {esql} rule generates, add them to the rule's  <<rule-ui-advanced-params,custom highlighted fields>>. 
+
+NOTE: New fields are fields that don't exist in the query’s source index.
```

Comment:
I would maybe add an example here.

```suggestion
NOTE: New fields are fields that don't exist in the query’s source index. For example, fields created by [eval command](https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-commands.html#esql-eval) or [rename](https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-commands.html#esql-rename)
```

Reply:
This is a good idea! I think one example would suffice here, and we could expand on it to like so:

```suggestion
NOTE: New fields are fields that don't exist in the query’s source index. For example, if you use the {ref}/esql-commands.html#esql-rename[`RENAME`] command to change the name of an existing field, and the renamed field doesn't exist in the query's source index, it's considered a new field. 
```
Let me know if the above is correct or if it needs to be tweaked at all.

Reply:
Can we give here example of `EVAL` command? It might be more common use case to use `EVAL`, rather than just rename field

Reply:
Added to draft via [ab386a5](https://github.com/elastic/security-docs/pull/5182/commits/ab386a53c89d2672b5490a212243d7666c4d98d2).